documentation/src/main/asciidoc/entity-view/manual/en_US/05_fetch_strategies.adoc

```diff
@@ -14,7 +14,7 @@ and only change the strategy on a case by case basis.
 
 === Join fetch strategy
 
-The `JOIN` strategy is not only the default, but also the one that makes mostly sense.
+If your JPA provider supports entity joins, the `JOIN` strategy is probably the strategy that makes mostly sense.
```

Comment:
Rephrase and simplify to:
"If your JPA provider supports entity joins, the `JOIN` strategy usually makes sense most of the time."